packages/rolldown/src/utils/error.ts

```diff
@@ -1,17 +1,21 @@
 import { type BindingError } from '../binding';
 import type { RollupError } from '../log/logging';
 
-export function normalizeErrors(rawErrors: BindingError[]): Error {
-  const errors = rawErrors.map((e) =>
-    e.type === 'JsError'
-      ? e.field0
-      // strip stacktrace of errors from native diagnostics
-      : Object.assign(new Error(), {
-        kind: e.field0.kind,
-        message: e.field0.message,
-        stack: undefined,
-      })
-  );
+function normalizeBindingError(e: BindingError): Error {
+  return e.type === 'JsError'
+    ? e.field0
+    : Object.assign(new Error(), {
+      kind: e.field0.kind,
+      message: e.field0.message,
+      stack: undefined,
+    });
+}
+
+export function aggregateBindingErrorsIntoError(
```

Comment:
Since you specify the `from` error type, you should probably also specify the `to` error type, like `JsError`

Reply:
Changed in #6389.